docs/api/ethereum.md

```diff
@@ -1640,7 +1648,11 @@ A class instance `variable` holding a `string` which represents an Ethereum acco
 When you are deploying a new ledger, this variable represents the ledger ID and is `null` until a mutation is completed.
```

Comment:
When you are deploying a new ledger, this variable represents the ledger ID and remains `null` until a mutation is completed.